src/ansys/geometry/core/sketch/arc.py

```diff
@@ -342,3 +345,123 @@ def from_three_points(cls, start: Point2D, inter: Point2D, end: Point2D):
 
         # Finally... you can create the arc
         return Arc(start=start, end=end, center=center, clockwise=is_clockwise)
+
+    @classmethod
+    @check_input_types
+    def from_start_end_and_radius(
+        cls,
+        start: Point2D,
+        end: Point2D,
+        radius: Union[Quantity, Distance, Real],
+        convex_arc: Optional[bool] = False,
+        clockwise: Optional[bool] = False,
+    ):
+        """
+        Create an arc from a starting point, an ending point, and a radius.
+
+        Parameters
+        ----------
+        start : Point2D
+            Starting point of the arc.
+        end : Point2D
+            Ending point of the arc.
+        radius : Union[Quantity, Distance, Real]
+            Radius of the arc.
+        convex_arc : bool, default: False
+            Whether the arc is convex or not. The default is ``False``.
+            When ``False``, the arc is concave.
+            When ``True``, the arc is convex.
```

Comment:
```suggestion
            Whether the arc is convex or not. The default is ``False``,
            in which case the arc is concave. If ``True``, the arc is convex.
```
I see you decided to remove the "default" portion. I'd now suggest searching and replacing to use the suggested text, which is how I would edit it to be consistent with other libraries.